targets/zephyr/Makefile.zephyr

```diff
@@ -116,21 +117,22 @@ endif
 	cmake -B$(INTERM) -H./ \
 	 -DENABLE_LTO=OFF \
 	 -DENABLE_VALGRIND=OFF \
+	 -DFEATURE_PROFILE=minimal \
 	 -DCMAKE_BUILD_TYPE=Release \
 	 -DCMAKE_VERBOSE_MAKEFILE=$(V) \
+	 -DMEM_HEAP_SIZE_KB=$(JERRYHEAP) \
 	 -DEXTERNAL_CMAKE_C_COMPILER=$(CC) \
 	 -DEXTERNAL_CMAKE_C_COMPILER_ID=GNU \
 	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=$(CPU) \
-	 -DEXTERNAL_MEM_HEAP_SIZE_KB=$(JERRYHEAP) \
 	 -DEXTERNAL_COMPILE_FLAGS="$(EXT_CFLAGS)" \
-	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=lakemont \
 	 -DEXTERNAL_LIBC_INTERFACE="$(ZEPHYR_LIBC_INC)" \
```

Comment:
There is no EXTERNAL_LIBC_INTERFACE option anymore. If you want to use an external libc make sure that the JERRY_LIBC and COMPILER_DEFAULT_LIBC options value are off and and add its include dir (ZEPHYR_LIBC_INC in this case) to the EXTERNAL_COMPILE_FLAGS value. I think its related to the problem you mentioned above in the TODO section.